content/kubernetes/re-clusters/redis-on-flash.md

```diff
@@ -0,0 +1,83 @@
+---
+Title: Use Redis on Flash on Kubernetes
+linkTitle: Redis on Flash
+description: Deploy and configure a Redis cluster or database on Kubernetes that supports Redis on Flash.
+weight: 16
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/re-clusters/redis-on-flash.md,
+    /kubernetes/re-clusters/redis-on-flash/,
+]  
+---
+
+## Prerequisites
+
+Redis Enterprise Software for Kubernetes supports using NVMe (nonvolatile memory express) SSDs (solid state drives) for flash storage with the Redis on Flash feature.
+
+Before creating your Redis clusters or databases, these SSDs must be:
```

Comment:
```suggestion
Before creating your Redis clusters or databases, your SSD or NVMe must be:
```

Reply:
> 

My understanding is the NVMe is a type of SSD.